### ISSUE TITLE
pictures Size

### ISSUE THREAD

**Issue by CyruS1337** [darickc/MMM-BackgroundSlideshow#15]
Hello

I wanted to ask what settings I need to make in your module so that my images are displayed on the PI as well as on the monitor on my computer. As you can see from the thread, my computer displays the pictures in the correct size, which is not the case with your module. Is there still a possibility that the images will be displayed correctly in your module? Because I love your module and do not want to switch to another.

Thank you already for your efforts!

**Comment by darickc** [comment]
It looks like you may have different values for the backgroundSize config options.  From the readme: 
cover: Resize the background image to cover the entire container, even if it has to stretch the image or cut a little bit off one of the edges.
contain: Resize the background image to make sure the image is fully visible

Monitor looks like it has contain and the pi has contain set.

**Comment by CyruS1337** [comment]
Somehow I overlooked that. So you mean I should play with the value "brackgroundSize":

The sizing of the background image. Values can be:
cover: Resize the background image to cover the entire container, even if it has to stretch the image or cut a little bit off one of the edges.
contain: Resize the background image to make sure the image is fully visible

Example: 'contain' 
Default value: 'cover' 
This value is OPTIONAL

**Comment by darickc** [comment]
Yes. That is correct.  If you want it to show the entire image and don't mind block bars if the image isn't the same aspect ratio as your monitor, use contain.  If you don't want any black bars and the image to fill the entire screen then cover is what you want.

What I did was resized and cropped all the landscape images (because I have my monitor in landscape) so they all fit 1920x1080 because I wanted it to fill the entire screen.  But my portrait ones I just resized.  I resized them so there is less memory it has to deal with and loads them faster.  No need to have a resolution that is 5x that of your monitor.  The I am using contain.  Since the landscape photos are the same resolution as the monitor there are no black lines.  But the portrait ones don't get cut off.
